components/eam/src/physics/cam/ndrop.F90

```diff
@@ -969,9 +969,10 @@ subroutine dropmixnuc( &
                  + raercol_cw(pver,mm,nsav)*(nact(pver,m) - taumix_internal_pver_inv)
             srcn(pver) = srcn(pver) + max(0.0_r8,tmpa)
          end do
+!! BJG 10/3 remove two arguments of "zero" to explmix here and elsewhere, since have no impact on output.
```

Comment:
I think comments regarding code restructuring belong better to Confluence pages unless they are important to understand the code. We can remove this comment.